cli/src/main.rs

```diff
@@ -624,32 +623,23 @@ fn verification_key<T: FieldElement>(
         .with_setup_file(params.map(PathBuf::from))
         .with_backend(*backend_type);
 
-    let vkey = pipeline.verification_key()?;
-
-    write_verification_key_to_fs(vkey, dir);
+    let mut vkey_file = BufWriter::new(fs::File::create(dir.join("vkey.bin")).unwrap());
+    pipeline.export_verification_key(&mut vkey_file)?;
+    vkey_file.flush().unwrap();
```

Comment:
a bit ugly to always have to do this flush

Reply:
This is just so write errors aren't ignored. I thought of two solutions for this:

1. a new BufWriter-like struct that panics if there is an error flushing inside `drop()` (instead of ignoring it);
2. a plain function that encapsulates this repeated code.

Reply:
Wrote a new function for these calls.